flexget/plugins/list/imdb_list.py

```diff
@@ -104,6 +104,8 @@ def items(self):
                                  (self.list_id, self.user_id))
             except HTTPError as e:
                 raise PluginError(e.args[0])
+            except ConnectionError as e:
```

Comment:
If it's the same handling as HTTPError wouldn't it be better to group those together?